media-sound/teamspeak-client/teamspeak-client-3.2.2.ebuild

```diff
@@ -18,8 +18,8 @@ SLOT="0"
 IUSE="alsa pulseaudio"
 REQUIRED_USE="|| ( alsa pulseaudio )"
 
-RDEPEND="app-arch/snappy:=
-	<dev-libs/openssl-1.1:0=
+RDEPEND="app-arch/snappy:0
```

Comment:
**Sub**slot, i.e. with the part after the slash.

Reply:
I am not sure, if this is so correct, could you have a lookt at my latest commit?